src/encoded/static/components/item-pages/components/AttributionTabView.js

```diff
@@ -4,11 +4,9 @@ import React from 'react';
 import _ from 'underscore';
 import PropTypes from 'prop-types';
 import ReactTooltip from 'react-tooltip';
-import { object } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
-import { ItemFooterRow } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/ItemFooterRow';
 import { FormattedInfoBlock, WrappedCollapsibleList } from './FormattedInfoBlock';
 import { Publications } from './Publications';
-
+import { ItemFooterRow, ExternalReferencesStackedTable } from './ItemFooterRow';
```

Comment:
Codacy found an issue: ['ExternalReferencesStackedTable' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5995039)